txnsync/msgorderingheap_test.go

```diff
@@ -218,7 +218,8 @@ func TestMultiThreaded(t *testing.T) {
 
 	}
 
-	for i := 0; i < loopCount; i++ {
+	startTime := time.Now()
+	for time.Now().Sub(startTime) < loopTime {
```

Comment:
nit: static check complains about it. I understand we use this statement everywhere but switching to Since would allow minimizing number of places to fix when we enable more linters.
```
should use time.Since instead of time.Now().Sub (S1012)go-staticcheck
```